design/sketch.webidl

```diff
@@ -428,14 +428,46 @@ interface WebGPURenderPipeline {
 // COMMAND RECORDING (Command buffer and all relevant structures)
 // ****************************************************************************
 
-enum WebGPULoadOp {
+/// Common interface for render and compute pass encoders.
+interface WebGPUCommandEncoder {
```

Comment:
nit: WebGPUProgrammablePassEncoder

Reply:
is there an intention/plan to have non-programmable passes at any point?

Reply:
* that they will read but won't need to type

Reply:
Actually I'm not even sure they will read it. I don't think `WebGLRenderingContextBase` even shows up when you're using WebGL.

I believe this would be done by using `interface mixin WebGPUProgrammablePassEncoder`.

Reply:
I'm with @kvark here - What does a "non-programmable pass" even mean?